docs/apm/traces/tracing-ingest.md

```diff
@@ -0,0 +1,59 @@
+---
+id: tracing-ingest
+title: Use Tracing to Troubleshoot Ingest
```

Comment:
I would change the title to just Troubleshooting or Troubleshooting Ingest (better in my opinion). Current title suggest we are saying how to use tracing (product) to troubleshoot customers' applications' ingest.

Reply:
Changed to "Troubleshoot Ingest".